ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -62,8 +63,21 @@ module.exports = Marionette.LayoutView.extend({
     this.$el.toggleClass('has-feature', this.model.get('target') !== undefined)
   },
   serializeData: function() {
+    const that = this
     let modelJSON = this.model.toJSON()
+    let summaryModel = {}
+    const summary = props.summaryShow
+    summary.forEach(function(attribute) {
+      if (that.model.get('targetMetacard') !== undefined)
```

Comment:
Need to double check if 'targetMetacard' grabs the hovered metacard in play or if it grabs only targetMetacard types.

Reply:
I think it's the right metacard you're looking for.